ui/app/AppLayouts/Onboarding2/OnboardingLayout.qml

```diff
@@ -148,22 +140,13 @@ Page {
             root.onboardingStore.setPin(pin)
         }
 
-        onShareUsageDataRequested: (enabled) => {
-            root.metricsStore.toggleCentralizedMetrics(enabled)
-            Global.addCentralizedMetricIfEnabled(
-                        "usage_data_shared",
-                        { placement: Constants.metricsEnablePlacement.onboarding })
-            localAppSettings.metricsPopupSeen = true
-        }
```

Comment:
Good move imo; this is mainly for @jrainville, I guess you'll want to handle this signal  yourself from now in main.qml